### PR TITLE
feat: support WordPress construct

### LINKED ISSUE #28: custom healthCheck for eatch task/service
We should allow each task to specify custom `healthCheck` from elbv2 target group to the service.

### PR DESCRIPTION
**Pull request by pahud**
Introducing the new `WordPress` construct in this PR that allows you to run a high available wordpress on AWS Fargate and Amazon Aurora Serverless.

Unit testing coming in another PR.

Fixes #28 #29 #30

### CODE REVIEW

**Comment by pahud** [comment]
@neilkuan

File: src/main.ts [hunk 1]
```diff
@@ -233,44 +246,52 @@ export class DualAlbFargateService extends cdk.Construct {
...
-      vpc: this.vpc,
-    });
 
-    if (this.hasInternalLoadBalancer) {
-      new route53.ARecord(this, 'InternalAlbAlias', {
-        zone,
-        recordName: internalAlbRecordName,
-        target: route53.RecordTarget.fromAlias(new targets.LoadBalancerTarget(this.internalAlb!)),
+    if (this.enableLoadBalancerAlias) {
+      console.log('ENABLE ALIAS');
```
Comment (neilkuan) [inline src/main.ts:257]: we need to remove this console log ?!

**Comment by pahud** [comment]
@neilkuan round 2

**Review by neilkuan — APPROVED** [review]
LGTM

**Pull request merged**